Website deploy script needs fixing

Our script for deploying to the website got broken.

```
Run ./scripts/deploy_website.sh
  ./scripts/deploy_website.sh
  shell: /bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.8.7/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.8.7/x64/lib
    TAG: 0.21.0
    WORKSPACE: /home/runner/work/cobrapy/cobrapy
    WEBSITE_DEPLOY_TOKEN: ***
./scripts/deploy_website.sh: line 5: TMP: unbound variable
Error: Process completed with exit code 1.
```

--- a/scripts/deploy_website.sh
+++ b/scripts/deploy_website.sh
@@ -2,7 +2,7 @@
 
 set -eu
 
-target="${TMP}/cobrapy-website"
+target="${HOME}/cobrapy-website"
 
 git clone "https://github.com/opencobra/cobrapy-website.git" "${target}"
 pushd "${target}"

